serverless/pytorch/omerferhatt/xmem/nuclio/function-gpu.yaml

```diff
@@ -0,0 +1,67 @@
+metadata:
+  name: pth-omerferhatt-xmem
+  namespace: cvat
+  annotations:
+    name: XMem
+    type: tracker
+    spec:
+    framework: pytorch
+
+spec:
+  description: Long-Term Object Segmentation with an Atkinson-Shiffrin Memory Model
+  runtime: 'python:3.9'
+  handler: main:handler
+  eventTimeout: 30s
+
+  env:
+    - name: PYTHONPATH
+      value: /opt/nuclio/xmem
+
+  build:
+    image: cvat.pth.omerferhatt.xmem:latest-gpu
+    baseImage: nvidia/cuda:12.1.0-runtime-ubuntu22.04
+
+    directives:
+      preCopy:
+        - kind: RUN
+          value: |-
+            apt update \
+              && apt install -y --no-install-recommends \
+                wget \
+                git \
+                ca-certificates \
+                python-is-python3 \
+                python3 \
+                python3-pip \
+              && rm -rf /var/lib/apt/lists/*
+        - kind: WORKDIR
+          value: /opt/nuclio
+        - kind: RUN
+          value: git clone --branch main https://github.com/omerferhatt/XMem xmem
+        - kind: RUN
+          value: pip install opencv-python-headless jsonpickle
+        - kind: RUN
+          value: |-
+            pip install torch torchvision
```

Comment:
Usually it is a good idea to fix dependencies version. Otherwise sometimes it will turn around, that the image cannot be build anymore.